services/worker-manager/test/api_test.js

```diff
@@ -548,8 +565,8 @@ helper.secrets.mockSuite(testing.suiteName(), ['db'], function(mock, skipping) {
   });
 
   test('get workers for a given worker pool - no workers', async function () {
-    const workerPoolId = 'r/r';
-
+    //const workerPoolId = 'r/r';
+    await createWorkerPool({providerId : 'google'});
```

Comment:
Does this not succeed if you just pass `{}` without a providerId?

Reply:
Yes.
I have made all the necessary changes.